docker-compose-dev.yml

```diff
@@ -203,24 +203,17 @@ services:
       - flow-api
 
   minio:
-    image: minio/minio:RELEASE.2024-11-07T00-52-20Z@sha256:ac591851803a79aee64bc37f66d77c56b0a4b6e12d9e5356380f4105510f2332
+    image: bitnami/minio:2025.6.13
     volumes:
       - minio-data:/data
     ports:
       - "9001:9000"
       - "9002:36987"
     environment:
-      <<: *minio-config
-    entrypoint: >
-      /bin/sh -c "
-      export MINIO_ROOT_USER=$$PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_ACCESS_KEY_ID;
-      export MINIO_ROOT_PASSWORD=$$PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_SECRET_ACCESS_KEY;
-      minio server /data --console-address=":36987" &
-      sleep 2;
-      /usr/bin/mc alias set myminiopeerdb http://minio:9000 $$MINIO_ROOT_USER $$MINIO_ROOT_PASSWORD;
-      /usr/bin/mc mb myminiopeerdb/$$PEERDB_CLICKHOUSE_AWS_S3_BUCKET_NAME;
-      wait
-      "
+      MINIO_ROOT_USER: _peerdb_minioadmin
```

Comment:
these should still be dependent on `PEERDB_` environment variables for people who configure these